scripts/prepdocs.ps1

```diff
@@ -52,12 +52,17 @@ if ($env:AZURE_TENANT_ID) {
   $tenantArg = "--tenantid $env:AZURE_TENANT_ID"
 }
 
+if ($env:USE_FEATURE_INT_VECTORIZATION) {
+  $integratedVectorizationArg = "--useIntVectorization $env:USE_FEATURE_INT_VECTORIZATION"
+}
+
 $argumentList = "./scripts/prepdocs.py $dataArg $adlsGen2StorageAccountArg $adlsGen2FilesystemArg $adlsGen2FilesystemPathArg $searchAnalyzerNameArg " + `
-"$aclArg --storageaccount $env:AZURE_STORAGE_ACCOUNT --container $env:AZURE_STORAGE_CONTAINER " + `
+"$aclArg --storageaccount $env:AZURE_STORAGE_ACCOUNT --container $env:AZURE_STORAGE_CONTAINER --storageresourcegroup $env:AZURE_STORAGE_RESOURCE_GROUP " + `
```

Comment:
I just made a change to master to reorganize these arguments to be in a more sensible order, so you'll need to pull and merge, sorry.